<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Humidity</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Temperature</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Maximum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Minimum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Average</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="analyse_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plot_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>282</width>
     <height>117</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="read_btn">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Humidity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="temp_out">
        <property name="styleSheet">
         <string notr="true">color: red;
background-color: lightgrey;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLCDNumber" name="hum_out">
        <property name="styleSheet">
         <string notr="true">color: red;
background-color: lightgrey;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>371</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Read Many</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="hum_alarm">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>humidity alarm: 60</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="temp_alarm">
        <property name="placeholderText">
         <string>Temperature alarm: 80</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="table">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Humidity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Temperature</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
